toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/AuthenticatorState.kt

```diff
@@ -330,8 +398,8 @@ private class AuthenticatorStateImpl(
      * @return a [Flow] with a [UsernamePassword] provided by the user, or null if the user cancelled.
      * @since 200.2.0
      */
-    private suspend fun awaitUsernamePassword(url: String, exception: Throwable? = null): Flow<UsernamePassword?> =
-        callbackFlow<UsernamePassword?> {
+    private fun awaitUsernamePassword(url: String, exception: Throwable? = null): Flow<UsernamePassword?> =
```

Comment:
Since this function doesn't suspend we might want to rename it, how about:
```suggestion
    private fun usernamePasswordFlow(url: String, exception: Throwable? = null): Flow<UsernamePassword?> =
```

Reply:
God point, updated it.